docs/Running-Mastodon/Production-guide.md

```diff
@@ -1 +0,0 @@
-[The documentation has moved to its own repository](https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Production-guide.md)
```

Comment:
I mean this PR kinda of has a lot of unrelated commits from your fork and the unrelated file change still records so could you remake it using just the GitHub UI?